extensions/amp-story/0.1/media-pool.js

```diff
@@ -53,6 +54,12 @@ const REPLACED_MEDIA_ATTRIBUTE = 'replaced-media';
 const DOM_MEDIA_ELEMENT_ID_PREFIX = 'i-amphtml-media-';
 
 
+/**
+ * @const {string}
+ */
+const POOL_MEDIA_ELEMENT_PROPERTY_NAME = 'i-amphtml-media-pool';
```

Comment:
This looks like a CSS class. AFAIK the convention for element props is `__AMP__FOO__`. See action-impl.js for an example.

Reply:
Done